app/models/concerns/type_of_application.rb

```diff
@@ -30,7 +30,7 @@ def post_submission_evidence?
   alias pse? post_submission_evidence?
 
   def appeal_no_changes?
-    return false unless kase
+    return false unless kase&.appeal_original_app_submitted == 'yes'
 
```

Comment:
This guard was added to catch when the appeal section is re-done and appeal_original_app_submitted is changed to 'no'